### PR TITLE
[stable/quassel] Update image and deployment apiVersion

### PR DESCRIPTION
**Pull request by WyriHaximus**
For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file: @kiall

#### What this PR does / why we need it:

Ensures that this chart works on the latest versions of Helm and update to the latest version of the used Docker image

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/quassel/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.12.4
```
Comment (davidkarlsen) [inline stable/quassel/Chart.yaml]: make this match too?

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/quassel/Chart.yaml (hunk 1 above)
Comment (WyriHaximus) [reply stable/quassel/Chart.yaml]: Ah yes my bad :zipper_mouth_face:

**Comment by WyriHaximus** [comment]
Is there a way I can help move this forward?

**Comment by WyriHaximus** [comment]
Bump

**Comment by WyriHaximus** [comment]
Bump

**Comment by WyriHaximus** [comment]
Bump

**Comment by WyriHaximus** [comment]
Bump

**Comment by WyriHaximus** [comment]
Bump

**Comment by WyriHaximus** [comment]
Bump

**Comment by WyriHaximus** [comment]
Bump

**Comment by WyriHaximus** [comment]
Bump

**Comment by davidkarlsen** [comment]
/assign
/lgtm
Sorry for the delay!

**Review by davidkarlsen — APPROVED** [review]

**Comment by WyriHaximus** [comment]
> Sorry for the delay!

No problem, I had a really good chat with `stale` 😋

**Pull request merged**